frontend/src/components/object/ObjectTag.vue

```diff
@@ -107,7 +110,8 @@ watch( [props, tsGetTagging, vsGetTagging], () => {
   </div>
   <div>
     <Button
-      v-if="editable"
+      v-if="editable &&
+        permissionStore.isObjectActionAllowed(props.objectId, getUserId, Permissions.UPDATE, props.objectId)"
```

Comment:
May be better to move this v-if up to the div instead of the Button so that we aren't rendering as many empty elements.